### ISSUE TITLE
Accessibility feedback - Design intent collaboration: CT Filters

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#11334]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Must
* Review the user flow on the first page with respect to the typeahead. I think I wasn't asking this question well during the meeting, so I'll try again writing it out. When I land on the GIBCT, I broadly think about taking these steps:
  1. I set my veteran status from the first dropdown
  1. I confirm my GI Bill benefit in the next two dropdowns
  1. I set the type of institution and how I want to take classes using the two radio groups
  1. I start typing into the typeahead search. Here is where I have problems. The typeahead if I understand correctly is going to listen for more keywords, zip codes, and fragments of proper names. This is very powerful! What happens when I click on one of the options? Right now clicking on a typeahead suggestion fires the submit event and moves me to the search results page. I never press the `Search` button, and that feels like breaking the user contract a little bit.
* Is the typeahead going to break the results apart into logical groups? If I search for `los angeles` I would consider that might return names of schools, on the job training, and possibly the city. I think there's an opportunity for logical lists inside the typeahead suggest with headings and some good organization.

## Should
* Add language to the search results filter bar letting users know that making a selection will automatically re-run the search.
* Decide if the new filter interaction will set focus on the number of results, or if focus will stay on the current form element. I won't make a call here; just something that should be thought through as you discuss how filters might evolve.

## Consider
* Making the tags title case. The all upper case letters can create challenges for screen reader pronunciation.
* What might be different about these tags if they're upgraded to clickable links in the future? Will the visual style be enhanced to show they're interactive?
* It might be worth adding an SR-only span Historically Black Colleges and Universities if the tag `HBCU` stays an abbreviation. That way screen readers can read out the full name.
* If you're exploring a sticky filter bar, let's talk about some HTML markup we can add to make it maximally usable. At a minimum would need a `tabindex="0"` on the container div so it can take keyboard focus for arrow key traversal. There might be other considerations, but that's a minimum. Also interested to see how this evolves on mobile.
* I was intrigued by the idea of adding filters to the top of the search results page as users toggled them on. It could be a nice way to announce updated search result numbers, and what was added to the filter "bar". I'm tacking on a screenshot from Ebay that I won't vouch for the accuracy of the markup, but captures the idea pretty nicely.

---
<img width="925" alt="Screen Shot 2020-07-16 at 4 44 39 PM" src="https://user-images.githubusercontent.com/934879/87725919-c3edb480-c783-11ea-9028-e6910b982ea9.png">

## Definition of Done
* Team responds to the ticket in comments or email
* Team schedules any requested follow up consults with accessibility specialist

**Comment by amyknox** [comment]
Thanks for all this great input @1Copenut ! We'll take a look at it as the design evolves and get back to you with questions.
